README.md

```diff
@@ -1 +1 @@
-# Push-2-Tweet
+#<a id="section0"></a> Push-2-Tweet
```

Comment:
Included in https://github.com/telefonicaid/push-2-tweet/commit/14d8a0dad4f2997d3bf65e3d74865b685a7f4778